config/sync/views.view.senior_leaders_directory_es.yml

```diff
@@ -499,75 +499,75 @@ display:
           field: search_api_relevance
           relationship: none
           group_type: group
-          admin_label: ''
+          admin_label: ""
           order: DESC
           exposed: false
           expose:
-            label: ''
+            label: ""
           plugin_id: search_api
         family_name:
           id: family_name
           table: search_api_index_individuals
           field: family_name
           relationship: none
           group_type: group
-          admin_label: ''
+          admin_label: ""
           order: ASC
           exposed: false
           expose:
-            label: ''
+            label: ""
           plugin_id: search_api
-      title: 'The Network of Senior Leaders: the Connect Directory'
+      title: "The Network Directory of Senior Leader"
```

Comment:
I think there's a typo here — 'Leader' is singular, but should be plural, right?